h/models/document/_document.py

```diff
@@ -153,12 +152,19 @@ def merge_documents(session, documents, updated=None):
 
     try:  # pylint:disable=too-many-try-statements
         session.flush()
-        session.query(Annotation).filter(
-            Annotation.document_id.in_(duplicate_ids)
-        ).update({Annotation.document_id: master.id}, synchronize_session="fetch")
+
+        # This service method was creating from code existing here with
```

Comment:
A bit of a wart but I reckon moving this to the service leaves it better than before.

Moving all of this to DocumentService should be part of another refactor PR.